shaders/src/shading_parameters.fs

```diff
@@ -33,7 +33,12 @@ void computeShadingParams() {
 #endif
 
     shading_position = vertex_worldPosition.xyz;
-    shading_view = normalize(frameUniforms.cameraPosition - shading_position);
+
+    // Perspective or orthographic:
+    shading_view = isPerspectiveProjection() ?
```

Comment:
I think I can live with this. Could you add a better comment that explains quickly why we need a different vector for ortho?

Reply:
Sure, done in 24a67b7d4b644e9ebb724807be04268a91a86583